### ISSUE TITLE
Validate `env` file

### ISSUE THREAD

**Issue by pablo-clueless** [CommunityPro/ecommerce-api#14]
- Create a function that validates all environment variables when the app starts.
- The function should only allow the app to start if all environment variables are verified to be present and of the required type
- An error should be thrown instead of the app crashing if a variable isn't present.

**Comment by The-CodeINN** [comment]
Is this resolved? If not, I'd like to work on this @pablo-clueless

**Comment by bnjox** [comment]
@The-CodeINN any updates on this??
still working on it..??

**Comment by The-CodeINN** [comment]
Yes, but I was unable to start the application. Here is my approach:

1.  Create an env file with the following content: 
 ```
APP_VERSION=v1
NODE_ENV=development
PORT=5000
JWT_SECRET=pfkkskwdlqoeka
```

2. Run `pnpm predev`
3. Run `pnpm dev`
4. No response when I tried to access it on `localhost:5000
`

Also i have a lot of generated files, adding to my changes:

![image](https://github.com/user-attachments/assets/cd0653a3-b70b-4f25-bf07-00a973cc34df)

**Comment by bnjox** [comment]
I think the generated files are from `vitepress`.

I'll give it a try and share my findings...

**Comment by bnjox** [comment]
@The-CodeINN I checked, and based on your steps, it needs to have a database URI before it can even run.
I opened an [issue](#70) that is probably related and would add it in the config file

**Comment by The-CodeINN** [comment]
Okay.. i think it's better we have a `.env.example` for reference

**Comment by pablo-clueless** [comment]
I'll add an `env.example` later today
